doc/semitrans.xml

```diff
@@ -251,3 +251,83 @@ Transformation( [ 7, 8, 8, 7, 8, 8, 8, 7, 8, 8 ] )]]></Example>
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="DigraphOfActionOnPairs">
+  <ManSection>
+    <Attr Name = "DigraphOfActionOnPairs" Arg = "coll" Label = "for ever" />
+    <Oper Name = "DigraphOfActionOnPairs" Arg = "coll[, n]" Label = "never" />
+    <Returns>A digraph.</Returns>
+    <Description>
+      If <A>coll</A> is a transformation collection and <A>n</A> is a
+      non-negative integer, then <C>DigraphOfActionOnPairs(<A>coll</A>,
+        <A>n</A>)</C> returns a digraph representing the <Ref Oper="OnSets"
+        BookName="ref"/> action of the transformations in <A>coll</A> on the
+      pairs of points in <C>[1 ..  <A>n</A>]</C>.  If the optional argument
+      <A>n</A> is not specified, then by default the value of
+      <C>DegreeOfTransformationCollection(<A>coll</A>)</C> will be chosen for
+      <A>n</A>.  <P/>
+
+      The <C>DigraphOfActionOnPairs</C> has <M><A>n</A> + {n \choose 2}</M>
+      vertices: the vertices <C>[1 .. <A>n</A>]</C> correspond to the points in
+      <C>[1 .. <A>n</A>]</C>, and the remaining vertices correspond to the pairs
+      of points in <C>[1 .. <A>n</A>]</C>. This correspondence is stored in
+      <Ref Oper="DigraphVertexLabels" BookName="Digraphs" />.
+      <P/>
+
+      The edges of <C>DigraphOfActionOnPairs</C> are defined as follows,
+      depending on whether or not <A>coll</A> is a semigroup. If <A>coll</A> is
+      a semigroup, define <C>gens</C> to be
+      <C>GeneratorsOfSemigroup(<A>coll</A>)</C>; otherwise define <C>gens =
+        <A>coll</A></C>.
+      <P/>
+
+      <List>
+        <Mark>Edges between pairs</Mark>
+        <Item>
+          For each two distinct pairs <C>{i, j}</C> and <C>{k, l}</C>, there is
+          an edge from the vertex corresponding to the first pair to the vertex
+          corresponding to the second pair if there exists a transformation
+          <C>f</C> in <C>gens</C> such that <C>OnSets([i, j]) = [k, l]</C>.  In
+          other words, there exists such an edge if some transformation in
+          <C>gens</C> maps the first pair to the second.
+        </Item>
+
+        <Mark>If <A>coll</A> is not a semigroup</Mark>
+        <Item>
+          For each pair <C>{i, j}</C> and for each point <C>k</C>, there is an
+          edge from the vertex corresponding to the pair to the vertex
+          corresponding to the point if there exists a transformation <C>f</C>
+          in <A>coll</A> such that <C>OnSets([i, j]) = [k]</C>. In other words,
```

Comment:
The paragraphs are similar but they don't repeat the same information. However the heading for the two similar paragraphs were incorrect.

Rather than the difference between the paragraphs being the case of semigroup/not semigroup, the difference is that the first paragraph describes edges between points, and the second describes edges from a pair to a point.

I've updated the headings in the documentation.